frontend/src/PagesAdmin/RecruitmentGangAllApplicantsAdminPage/RecruitmentGangAllApplicantsAdminPage.tsx

```diff
@@ -0,0 +1,144 @@
+
+export function RecruitmentGangAllApplicantsAdminPage() {
+  const { recruitmentId, gangId } = useParams();
+  const [recruitment, setRecruitment] = useState<RecruitmentDto>();
+  const [gang, setGang] = useState<GangDto>();
+  const [applications, setApplications] = useState<RecruitmentApplicationDto[]>([]);
+  const [showSpinner, setShowSpinner] = useState<boolean>(true);
+  const navigate = useCustomNavigate();
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    if (recruitmentId && gangId) {
+      getRecruitmentApplicationsForGang(recruitmentId, gangId)
+        .then((response) => {
+          setApplications(response.data);
+          setShowSpinner(false);
+        })
+        .catch((error) => {
+          toast.error(t(KEY.common_something_went_wrong));
+          console.error(error);
+        });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [recruitmentId, gangId]);
+
+  useEffect(() => {
+    if (gangId) {
+      getGang(gangId)
+        .then((gang) => {
+          setGang(gang);
+        })
+        .catch((data) => {
+          if (data.request.status === STATUS.HTTP_404_NOT_FOUND) {
+            navigate({ url: ROUTES.frontend.admin_gangs });
+          }
+          toast.error(t(KEY.common_something_went_wrong));
+        });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [gangId]);
+
+  useEffect(() => {
+    if (recruitmentId) {
+      getRecruitment(recruitmentId)
+        .then((resp) => {
+          setRecruitment(resp.data);
+        })
+        .catch((data) => {
+          // TODO add error pop up message?
+          if (data.request.status === STATUS.HTTP_404_NOT_FOUND) {
+            navigate({ url: ROUTES.frontend.admin_recruitment });
+          }
+          toast.error(t(KEY.common_something_went_wrong));
+          console.error(data);
+        });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [recruitmentId]);
+
+  const tableColumns = [
+    { content: t(KEY.recruitment_applicant), sortable: true },
+    { content: t(KEY.common_phonenumber), sortable: true },
+    { content: t(KEY.common_email), sortable: true },
+    { content: t(KEY.recruitment_position), sortable: true },
+    { content: t(KEY.recruitment_interview_time), sortable: true },
+    { content: t(KEY.recruitment_interview_location), sortable: true },
+    { content: t(KEY.recruitment_recruiter_status), sortable: true },
+  ];
+
+  const data = applications.map(function (application) {
+    const applicationURL = reverse({
+      pattern: ROUTES.frontend.admin_recruitment_applicant,
+      urlParams: {
+        applicationID: application.id,
+      },
+    });
+
+    return [
+      {
+        content: (
+          <Link url={applicationURL}>
+            {application.user.first_name} {application.user.last_name}
+          </Link>
+        ),
+      },
+      application.user.email,
+      application.user?.phone_number,
```

Comment:
Epost står under "Telefonnummer" atm, flip for å fikse

```suggestion
      application.user?.phone_number,
      application.user.email,
```

Reply:
Hmmm ja det var leit å se
Fikset opp i det nå